drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -23,6 +23,7 @@ local seq_num = 0
 
 local SHARED_KEY = "__shared_key"
 local CLOUD_PUBLIC_KEY = "__cloud_public_key"
+local SUPPORTED_ALARM_VALUES = { "damaged", "forcedOpeningAttempt", "unableToLockTheDoor", "notClosedForALongTime", "highTemperature", "attemptsExceeded" }
```

Comment:
Should "clear" be in here as well?

Reply:
actually the `clear` is hidden intentionally. lockAlarm capability doesn't represent as state. - e.g. highTemerapture, damaged can be occurs together - so this attribute works as an event.
and in some cases, the `clear` event is could be ambiguous for users - the damage fixed?, no one trying forced opening attempt anymore?.

So the `clear` won't generate push notification. and hidden at history for this device.